modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -53,7 +53,9 @@ import {{invokerPackage}}.auth.OAuth;
 
 public class ApiClient {
 
-    private String basePath = "{{{basePath}}}";
+    private String scheme = "{{scheme}}";
```

Comment:
Shouldn't this provide a default for "http" in a similar way that host does with "localhost"

Reply:
If I understand how this works correctly, then I believe this defaults to `https` if a scheme isn't specified (https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java#L98).

Reply:
Yes. that is correct. Thanks for pointing that out.